### ISSUE TITLE
Softmax applied to predictions in Chapter 4 and 5?

### ISSUE THREAD

**Issue by OldPatrick** [StatQuest/signa#18]
Hi Josh,

In the Notebook in Chapter 4 / 5 we are encouraged to use the CrossEntropyLoss, which itself applies a Softmax to the output.
However, looking at the tensors, I see values between above 1 and below 0, whereas the described Softmax should have values between 0 and 1. Am I missing something?

<img width="279" alt="Image" src="https://github.com/user-attachments/assets/f7b809cd-e310-41dc-adb0-6560191959d4" />

Also, it is stated to apply a manual softmax to the output in the initial init. Applying a torch or nn Softmax changes the predicted 'probabilites' of the class drastically:

<img width="367" alt="Image" src="https://github.com/user-attachments/assets/9b1de25b-7d35-4d65-a581-63fa933335aa" />

Can you help me out ?

**Comment by StatQuest** [comment]
The tensors going into the CrossEntropyLoss function will be un-normalized,
and thus, can go from -infinity to +infinity. The tensors that come out of
CrossEntropyLoss can go from 0 to positive infinity. It is only within the
actual call to CrossEntropyLoss that the softmax is applied, so, it is only
within that function that you will have tensors that are normalized between
0 and 1. So the question is, are the tensors you are printing out from
within that function? If so, how did you manage that?
